src/Factory.php

```diff
@@ -47,7 +47,7 @@ public function tryRelativizePath(string $path): string
             return $traceRenderer->tryRelativizePath($matches[0]);
         }, $trace);
         // echo (new Exception($msg))->getHtml();
-        'trigger_error'($msg . (!class_exists(\PHPUnit\Framework\Test::class, false) ? "\n" . $trace : ''), E_USER_DEPRECATED);
+        trigger_error($msg . (!class_exists(\PHPUnit\Framework\Test::class, false) ? "\n" . $trace : ''), E_USER_DEPRECATED);
```

Comment:
this is intented for some reason

Reply:
but why add code needed for test in production code?
i see some here and there and is wrong

Reply:
to allow smother migration between versions

lifecycle:
- old feature
- deprecate
- drop completely